tests/virt/utils.py

```diff
@@ -309,13 +310,21 @@ def validate_machine_type(vm, expected_machine_type):
     vm_machine_type = vm.instance.spec.template.spec.domain.machine.type
     vmi_machine_type = vm.vmi.instance.spec.domain.machine.type
 
+    # Workaround for s390x (https://github.com/kubevirt/kubevirt/issues/14953), as machine type missing in config and
+    # hardcoded to s390_ccw_virtio in kubevirt code. So incase of s390x machine_type in VM/VMI are different
+    # than one in libvirt xml
+    if expected_machine_type == MachineTypesNames.s390_ccw_virtio:
+        expected_libvirt_machine_type = MachineTypesNames.s390_ccw_virtio_rhel9_6
+    else:
+        expected_libvirt_machine_type = expected_machine_type
```

Comment:
if the machine_type is hardcoded in kubevirt and does not work correctly in s390x - I would prefer to skip this test in s390x environment, instead of the using some weird workarounds